pages/en/about/get-involved/index.md

```diff
@@ -16,11 +18,18 @@ layout: about
 - The official Node.js Twitter account is [nodejs](https://twitter.com/nodejs).
 - The [Node.js project calendar](https://nodejs.org/calendar) with all public team meetings.
 
-## Learning
+## Learning Materials
```

Comment:
https://github.com/search?q=nodejs.org%2Fen%2Fabout%2Fget-involved%23learning

I couldn't find anything on GitHub, so I'm not sure it will have much impact.

Reply:
thanks for checking!